dev_docs/RELEASES.md

```diff
@@ -9,34 +9,17 @@ For patch releases, only the version on the existing major and minor version bra
 
 ## Feature Freeze
 
-* For patch releases, ensure all relevant backport PRs are merged.
-  We use backport labels on PRs and automation to ensure labels are set.
+* For **patch releases**:
+  * ensure all relevant backport PRs are merged. We use backport labels on PRs and automation to ensure labels are set.
 
-* Update Changelog
-
-  * Review existing [changelogs/head](https://github.com/elastic/apm-server/tree/main/changelogs/head.asciidoc) to ensure all relevant notes have been added.
-  * Move changelog entries from _head_ to _release_version_:
-    * Minor version:
-      Create new changelog file from [changelogs/head.asciidoc](https://github.com/elastic/apm-server/blob/main/changelogs/head.asciidoc)
-      If changes should not be backported, keep them in the _changelogs/head.asciidoc_ file.
-      Don't forget to `include` and link to the new file in [main changelog](https://github.com/elastic/apm-server/blob/main/CHANGELOG.asciidoc) and the [release notes](https://github.com/elastic/apm-server/blob/main/docs/release-notes.asciidoc) file. [(Sample PR)](https://github.com/elastic/apm-server/pull/7956/files)
-    * Patch version: Add a new section to existing release notes. ([Sample PR](https://github.com/elastic/apm-server/pull/8313/files))
-  * Add `@elastic/obs-docs` as a reviewer.
 
 ## Day after Feature Freeze
 
-* For minor releases, cut a new release branch from `main` and update them.
-  * Release branch:
-    Update versions and ensure that the `BEATS_VERSION` in the Makefile is updated,
-    e.g. [#2803](https://github.com/elastic/apm-server/pull/2803/files).
-    Trigger a new beats update, once the beats branch is also created.
-    Remove the [changelogs/head.asciidoc](https://github.com/elastic/apm-server/blob/main/changelogs/head.asciidoc) file from the release branch.
-
-  * Main branch:
-    Update [.mergify.yml](https://github.com/elastic/apm-server/blob/main/.mergify.yml) with a new backport rule for the next version,
-    and update versions to next minor version, e.g. [#2804](https://github.com/elastic/apm-server/pull/2804).
-
-  The release manager will ping the teams, but you can already prepare this in advance on the day after Feature Freeze.
+* Trigger release workflow manually
+  * For **patch releases**: run the [`run-patch-release`](https://github.com/elastic/apm-server/actions/workflows/run-patch-release.yml) workflow.
```

Comment:
Describe what patch release automation does. It does not include changelog changes. It updates some versions iirc.

Reply:
For patch release notes we could add something like this:

* For patch releases, release notes must be manually added:
  * Add a new section to the existing release notes file ([Sample PR](https://github.com/elastic/apm-server/pull/12680)).
  * Review the [changelogs/head](https://github.com/elastic/apm-server/tree/main/changelogs/head.asciidoc) file and move all relevant release notes from `head.asciidoc` to `release_version.asciidoc`. If changes do not apply to the version being released, keep them in the `head.asciidoc` file.
  * Review the commits in the release to ensure all changes are reflected in the release notes.
  * Add your PR to the documentation release issue ([Sample Issue](https://github.com/elastic/dev/issues/2485)).

Reply:
I added the suggested changes to [`837bc9a` (#12983)](https://github.com/elastic/apm-server/pull/12983/commits/837bc9ab1169fbc63d6e9b2665060d276f7a6a5e)